appserver/admingui/payara-console-extras/src/main/resources/asadminRecorder/asadminRecorderServer.jsf

```diff
@@ -86,6 +107,9 @@
     <sun:property id="filteredCommandsProp" labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nar.asadminRecorder.configuration.filteredCommands}"  helpText="$resource{i18nar.asadminRecorder.configuration.filteredCommandsHelp}">
         <sun:textField id="filteredCommands" columns="$int{75}" maxLength="255" text="#{pageSession.valueMap['filteredCommands']}" />
     </sun:property>
+    <sun:property id="passwordFileProp" labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nar.asadminRecorder.configuration.passwordFile}"  helpText="$resource{i18nar.asadminRecorder.configuration.passwordFileHelp}">
+        <sun:textField id="passwordFile" columns="$int{75}" maxLength="255" text="#{pageSession.valueMap['passwordFile']}" />
```

Comment:
Needs to match the value in the extraProperties map returned by the Get command, which is "prependPasswordFile"